### PR TITLE
worked on purpose field into a text area

### PR DESCRIPTION
**Pull request by ToBuild-2021**
Please go through my task.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (swarajpure) [inline .vscode/settings.json:3]: Please revert the changes unrelated to this PR

Comment (ToBuild-2021) [reply .vscode/settings.json]: Please check PR .

File: index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
       </div>
       <div class="inputBox">
         <label for="purpose">Purpose:</label>
-        <input type="text" id="purpose" name="purpose" placeholder="Purpose">
+        <textarea class="textinput"  id="purpose" name="purpose" placeholder="Purpose..."></textarea>
```
[the remark this answers is not in the document]
Comment (ToBuild-2021) [reply index.html]: Done with changes please check .

**Comment by PremanandChowdhury** [comment]
LGTM 👍🏼

**Review by PremanandChowdhury — APPROVED** [review]
I am approving the PR from my side 👌🏼

**Comment by ToBuild-2021** [comment]
> I am approving the PR from my side 👌🏼

Thank You!
